src/gui/shutdownconfirmdlg.cpp

```diff
@@ -30,20 +30,22 @@
  * Contact : hammered999@gmail.com
  */
 
-#include <QVBoxLayout>
-#include <QHBoxLayout>
+#include "shutdownconfirmdlg.h"
+#include "ui_shutdownconfirmdlg.h"
```

Comment:
Let us define the order of includes.
1. Module header (in .cpp)
2. System/Qt/Boost etc. headers (splitted in subcategories if you have many).
3. Application headers, starting from `base/` headers.

@sledgehammer999 what do you think about this? Should we have any rules about this?